Firestore/Example/Tests/Integration/API/FIRCursorTests.mm

```diff
@@ -192,4 +192,82 @@ - (void)testCanBeUsedInDescendingQueries {
   XCTAssertEqualObjects(FIRQuerySnapshotGetData(snapshot), (@[ @{ @"v" : @"d", @"sort" : @3.0 } ]));
 }
 
+FIRTimestamp *timestampWithMicros(int64_t seconds, int32_t micros) {
```

Comment:
Functions should have an initial capital.

Reply:
Done.